src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -866,6 +870,32 @@ Prevents content overflow in navbar. */
     padding: 20px 40px;
 }
 
+#student-data-spreadsheet .panel-body {
+    padding: 5px;
+    margin: 10px;
+}
+
+#enrollstudents {
+    max-width: 100%;
+    opacity: 0;
+    height: 0;
+    margin: 0;
+    padding: 0;
+    outline: 0;
```

Comment:
Is the purpose of all these rules just to hide the `textarea`? If so, it can be done by just `display: none;`

Reply:
Initially I did this and it failed `InstructorCourseEnrollPageUiTest`. This is because the `enroll` function in this class uses a function named `fillTextBox` which requires the textarea to be clickable. Therefore the textarea has to be "visible" to the system.

The `enroll` function is common among other test classes so I didn't modify it and used this workaround instead.

Reply:
If we replace the old textbox with the spreadsheet interface, `InstructorCourseEnrollPage.enroll` needs to be reimplemented to fill in the spreadsheet rather than using `fillTextBox`.

Reply:
I pondered over this a little and here are some of my thoughts. 

Since the form submission uses data in the textbox, why is there a need to fill in the spreadsheet for the test? From my point of view, the spreadsheet interface is used more for the user experience. 

However, I feel that the browser test is used to check if if the process of enrolling students is verified. Moreover, there is still a need to fill in the textbox anyway for form submission even if we implement functionality for filling in the spreadsheet during the test.

Reply:
> the spreadsheet interface is used more for the user experience and the browser test is used to check if the process of enrolling students is verified

Browser tests are supposed to mimic user's actions. That's why we need to use Selenium to control a browser. If it's just for testing the server's behaviour, we can just send the required HTTP request directly to the server.

> Moreover, there is still a need to fill in the textbox anyway for form submission even if we implement functionality for filling in the spreadsheet during the test.

Filling the textbox is to be done by the Javascript that you wrote, which will be tested here. Omitting some steps because they "should be done anyway" is not the right approach for end-to-end testing, which is what the browser tests are for.